python/util/rights.py

```diff
@@ -29,7 +29,7 @@ def to_dict(self):
                 return_codes.append(code)
         return return_codes
 
-    def from_json(self, right_codes):
+    def from_json(self, right_codes, makes_dirty=True):
```

Comment:
looks like a param which has never been actually used.

Reply:
it's not used in this definition, but you want the prototypes of `from_json` to be the same.

Reply:
got it :+1: